easybuild/easyconfigs/g/GMP/GMP-6.1.0-CrayGNU-2015.11-XC.eb

```diff
@@ -0,0 +1,24 @@
+# contributed by Luca Marsella (CSCS)
+easyblock = 'ConfigureMake'
+
+name = 'GMP'
+version = '6.1.0'
+
+homepage = 'http://gmplib.org/'
+description = """GMP is a free library for arbitrary precision arithmetic, 
+operating on signed integers, rational numbers, and floating point numbers. """
+
+toolchain = {'name': 'CrayGNU', 'version': '2015.11-XC'}
+
+sources = [SOURCELOWER_TAR_BZ2]
+source_urls = [GNU_SOURCE]
+
+preconfigopts = 'export CFLAGS=-mcmodel=large && '
```

Comment:
Sure, thank you, I forgot to insert $CFLAGS in the preconfigopts, as CFLAGS was likely undefined